src/language/collections.md

```diff
@@ -293,6 +293,12 @@ to create a list with three or four items in it:
 var nav = ['Home', 'Furniture', 'Plants', if (promoActive) 'Outlet'];
 ```
 
+Dart also supports [if-case][] inside collection literals:
+
+```dart
+var nav = ['Home', 'Furniture', 'Plants', if (promoActive case String x) $x];
```

Comment:
what is `promoActive` in this example? I cannot guess what this code does by looking at it!

Reply:
oh is the idea that `promoActive` would be a `String?` and this is equivalent to `if (promoActive is String) promoActive`?

What is the `$` on `$x`?

I find this example very confusing.

Reply:
I just grabbed this example from the [existing collection literals content](https://dart.dev/language/collections#control-flow-operators), which also doesn't give any context.

> What is the `$` on `$x`?

Ahh, idk! Thanks for calling this out, I'll come up with something clearer 🤞 (or hopefully someone else caught it and suggested a better example :))

Reply:
Changed it to:
```dart
var nav = ['Home', 'Furniture', 'Plants', if (login case 'Manager') 'Inventory'];
```
Any better?